### ISSUE TITLE
Altare Aenigmatica Client Crash

### ISSUE THREAD

**Issue by Micromith** [Arcaratus/BloodArsenal#115]
Hello!

It seems that the Altare Aenigmatica will cause a client crash when a player attempts to place any item into the block. Testing with all 5 blood orbs and a block of stone results in the same error. Although I will still include the individual crash reports.

I am using the FTB Beyond modpack Version: 1.11.0
                                           Minecraft Version: 1.10.2
                                      Blood Magic Version: 1.10.2-2.1.11-80
                                    Blood Arsenal Version: 1.10.2-2.1.0-22

[Weak Blood Orb Crash](https://github.com/Arcaratus/BloodArsenal/files/1640818/Blood.Arsenal.Crash.Report.txt)
[Apprentice Blood Orb Crash](https://github.com/Arcaratus/BloodArsenal/files/1640842/Apprentice.Blood.Orb.Crash.txt)
[Magicians Blood Orb Crash](https://github.com/Arcaratus/BloodArsenal/files/1640860/Magicians.Blood.Orb.Crash.txt)
[Archmage Blood Orb](https://github.com/Arcaratus/BloodArsenal/files/1640876/Archmage.Blood.Orb.txt)
[Trancendant Blood Orb](https://github.com/Arcaratus/BloodArsenal/files/1640884/Trancendant.Blood.Orb.txt)
[Stone Block](https://github.com/Arcaratus/BloodArsenal/files/1640902/Stone.Block.txt)

Some Prereading
Description: Updating Screen Events

Happy Coding,
                  
                         Micromith

**Comment by Boolyman** [comment]
I'm getting this crash too... can we get a response to this please.

**Comment by MachinegunMarty** [comment]
Can confirm that it happens to me to.

BloodArsenal-1.12.2-2.1.0-22
BloodMagic-1.12.2-2.2.12-97
Forge 1.12.2-2707

I placed an Altare Aenigmatica on the altar and at first it didn't work (Top, nor sides)

Then i placed some blocks on the altar (2x) and then got rid of the bottom block and placed the Altare Aenigmatica on the bottom side of the block so the spiral side was pointing towards the blood altar.
I clicked the Altare Aenigmatica and my client crashed and my server was spamming the console so fast i couldn't read what was going on there.

Had to reboot both my server and client. Client starts but my server keeps spamming the same message  and doesn't start anymore......

The crash from the server log:

[Server thread/FATAL] [FML]: ---- Minecraft Crash Report ----
// Why did you do that?

Time: 2018-06-18 15:35:07 CEST
Description: Ticking block entity

java.lang.ClassCastException: WayofTime.bloodmagic.tile.TileAltar cannot be cast to arcaratus.bloodarsenal.tile.TileInventory
    at arcaratus.bloodarsenal.tile.TileAltareAenigmatica.update(TileAltareAenigmatica.java:91)
    at net.minecraft.world.World.redirect$tileEntityUpdate$zzo000(World.java:5128)
    at net.minecraft.world.World.updateEntities(World.java:1832)
    at net.minecraft.world.WorldServer.updateEntities(WorldServer.java:613)
    at net.minecraft.server.MinecraftServer.updateTimeLightAndEntities(MinecraftServer.java:767)
    at net.minecraft.server.dedicated.DedicatedServer.updateTimeLightAndEntities(DedicatedServer.java:396)
    at net.minecraft.server.MinecraftServer.tick(MinecraftServer.java:668)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)
    at java.lang.Thread.run(Unknown Source)


I attached both log files for you:

Client log: 
[2018-06-18-4.log](https://github.com/Arcaratus/BloodArsenal/files/2111443/2018-06-18-4.log)

Server Log:
[latest.log](https://github.com/Arcaratus/BloodArsenal/files/2111450/latest.log)

I hope it helps you locate the problem

Thanx in advance !

grtz,

MachinegunMarty
